src/modules/batch-notifications/config/returns/lib/create-notification-data.js

```diff
@@ -99,7 +98,7 @@ const templateRandomiser = templateType => {
 
 const getRelevantRowData = (rows, reminderRef) => {
   const contactAndMessageType = reminderRef.substring(17) // remove 'returns_invitation' from beginning
-  return first(rows.filter(row => row.message_ref.includes(contactAndMessageType)))
+  return (rows.filter(row => row.message_ref.includes(contactAndMessageType)))[0]
```

Comment:
We've split this kind of thing across two lines elsewhere, it might be clearer to do it here as well? eg:
```suggestion
  const filteredRows = rows.filter((row) => row.message_ref.includes(contactAndMessageType))
  return filteredRows[0]
```